geode-core/src/main/java/org/apache/geode/internal/cache/LocalRegion.java

```diff
@@ -498,6 +498,8 @@ public StoppableCountDownLatch getInitializationLatchAfterGetInitialImage() {
    */
   private final Lock clientMetaDataLock = new ReentrantLock();
 
+  private boolean detectReadConflicts;
```

Comment:
I think this field should have been added to AbstractRegion instead of LocalRegion.
I think all the fields used to store region attributes are up on it.
Also make this field "final" unless we end up wanting to support changing it on  an existing region